packages/api-v2/src/AuthProvider.tsx

```diff
@@ -92,7 +92,7 @@ const AuthProvider = ({ children }: AuthProviderProps) => {
                 setData(res);
                 setIsLoading(false);
 
-                localStorage.setItem('active_loginid', newLoginid);
+                localStorage.setItem(customLoginIDKey ?? 'active_loginid', newLoginid);
```

Comment:
does getActiveLoginIDFromLocalStorage require the same change?
it feels that we're loading account id from "standard" key but saving to custom?

Reply:
I mean the use of getActiveLoginIDFromLocalStorage() within AuthProvider within line 29

Reply:
Hmm I think you're right, have made changes